remoteappmanager/handlers/home_handler.py

```diff
@@ -254,7 +252,6 @@ def _start_container(self, orm_user, app, policy, mapping_id):
         Container
         """
 
-        user_name = orm_user.name
```

Comment:
Ok, what's the point of an opaque object if you can't basically inquire anything about its state? How do we plan to use it?